packages/core/src/components/fields/ArrayField.tsx

```diff
@@ -863,7 +903,7 @@ class ArrayField<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends For
           title={title}
           index={index}
           schema={itemSchema}
-          uiSchema={itemUiSchema}
+          uiSchema={itemUiSchema || {}}
```

Comment:
Why add the `|| {}` when undefined works just fine?

Reply:
Fixed.